### PR TITLE
Theme: add color variable in variable file.

### PR DESCRIPTION
**Pull request by nongpp**
I am following your advice from  #8141 to initial variable in variable file and comment the color's name for easy to understand and change.

### CODE REVIEW

**Review by nschonni — APPROVED** [review]
Not sure if adding the colour in a comment helps, but not a big deal.
If someone lands this, they should squash and maybe give a more descriptive commit message

**Comment by LaurentGoderre** [comment]
@EricDunsworth an you verify the all the colors match? If so then this is good to merge.

It would be good to check if there is a way to link some of these variables together in a separate PR

**Comment by nongpp** [comment]
Yes, I verify all the colors match in Theme.

File: theme/_variables.scss [hunk 1]
```diff
@@ -12,4 +12,58 @@ $main-bg: $breadcrumb-bg; // Main element's background colour (WET override).
...
 
+/*Banner*/
+$wb-bnr-background-color: #114f7a; /*Dark Cerulean*/
+$wb-bnr-background-linear-gradient: linear-gradient(#176ca7, #114f7a); /*Lochmara, Dark Cerulean*/
+$wb-bar-background: #474747; /*Charcoal*/
+$wb-bnr-border-bottom: #fff; /*white*/
+$wb-bnr-color: #fff; /*white*/
+
+$wb-lng-glb-mn--pnl-btn-border-left: #777; /*gray*/
+$wb-lng-glb-mn--pnl-btn-a-color: #fff; /*white*/
```
Comment (EricDunsworth) [inline theme/_variables.scss]: These 2 variables contain two consecutive dashes between ``mn`` and ``pnl``. I'd recommend revising those variables' names to only use a single dash at that spot.

Specifically:

* ``$wb-lng-glb-mn--pnl-btn-border-left`` would become ``$wb-lng-glb-mn-pnl-btn-border-left``
* ``$wb-lng-glb-mn--pnl-btn-a-color`` would become ``$wb-lng-glb-mn-pnl-btn-a-color``

File: theme/banner/_base.scss [hunk 2]
```diff
@@ -26,7 +27,7 @@
 	margin: 0;
 
 	li {
-		border-left: 1px solid #777;
+		border-left: 1px solid $wb-lng-glb-mn--pnl-btn-border-left;
```
Comment (EricDunsworth) [inline theme/banner/_base.scss]: Reference to ``$wb-lng-glb-mn--pnl-btn-border-left`` should be renamed to ``$wb-lng-glb-mn-pnl-btn-border-left``.

File: theme/banner/_base.scss [hunk 3]
```diff
@@ -37,7 +38,7 @@
 	}
 
 	a {
-		color: #fff;
+		color: $wb-lng-glb-mn--pnl-btn-a-color;
```
Comment (EricDunsworth) [inline theme/banner/_base.scss]: Reference to ``$wb-lng-glb-mn--pnl-btn-a-color`` should be renamed to ``$wb-lng-glb-mn-pnl-btn-a-color``.

**Review by EricDunsworth — CHANGES_REQUESTED** [review]
A couple of variables contain multiple consecutive dashes (``--``) in their names. I'd recommend renaming them to only use a single dash in those situations.

**Comment by EricDunsworth** [comment]
@LaurentGoderre I just reviewed all the colour variables. They all match. Btw each variable is currently referenced only once.

Btw the capitalization in the comments is inconsistent. Some comments start off with capital letters, whereas others start with lowercase letters. I didn't bring it up in my review since that's pretty minor and isn't really a show-stopper.

File: theme/_variables.scss (hunk 1 above)
Comment (nongpp) [reply theme/_variables.scss]: @EricDunsworth Thank you for your advice. I will correct it.

**Comment by nongpp** [comment]
@EricDunsworth I had already change some code followed by your advice.

**Review by EricDunsworth — APPROVED** [review]
This PR's recent updates look good to me.

**Comment by LaurentGoderre** [comment]
@nongpp want to learn how to rebase yourself or do you want us to do it?

**Comment by nongpp** [comment]
@LaurentGoderre Sorry, I do not understand "learn how to rebase". Could you explain it for me?

**Comment by LaurentGoderre** [comment]
You have 9 commits to do one changeset. Did you use the Github UI to make the changes?

**Comment by nongpp** [comment]
Yes, I use the Github UI.

**Comment by nongpp** [comment]
@LaurentGoderre Thank you very much.

**Pull request merged**